memdown.js

```diff
@@ -2,12 +2,12 @@ var inherits          = require('inherits')
   , AbstractLevelDOWN = require('abstract-leveldown').AbstractLevelDOWN
   , AbstractIterator  = require('abstract-leveldown').AbstractIterator
   , ltgt              = require('ltgt')
-  , setImmediate      = global.setImmediate || process.nextTick
+  , setImmediate      = require('immediate')
   , createRBT = require('functional-red-black-tree')
   , globalStore       = {}
 
 function toKey (key) {
-  return typeof key == 'string' ? '$' + key : JSON.stringify(key)
+  return '$' + key
```

Comment:
Nope, this is actually only used for database names. So you'd no longer be able to do `new MemDOWN({'foo': 'bar'})` although I doubt anyone was doing that.